modules/hostap/src/supp_api.c

```diff
@@ -412,6 +414,15 @@ static int wpas_add_and_config_network(struct wpa_supplicant *wpa_s,
 				}
 			}
 
+			if (params->security == WIFI_SECURITY_TYPE_SAE_H2E ||
+			    params->security == WIFI_SECURITY_TYPE_SAE_AUTO) {
+				params->sae_pwe =
```

Comment:
This breaks the build, there is no `sae_pwe` in the params structure, this should be a local variable. Sorry, missed this in the review.

Reply:
Should I raise a urge pr to fix it?

Reply:
Already did https://github.com/zephyrproject-rtos/zephyr/pull/77274